community/cloud-foundation/templates/gcs_bucket/examples/gcs_bucket_iam_bindings.yaml

```diff
@@ -0,0 +1,31 @@
+# Example on how to use the cloud storage bucket template
+#
+# In this example, a storage bucket is created in location us-east1
+# with name test-bucket. Cloud storage bucket names should be unique globally
+# refer https://cloud.google.com/storage/docs/naming for bucket and object 
+# naming guidelines.
+#
+# IAM Policies are also applied with the `bindings` section
+# Replace the following with appropirate values
+# <FIXME:user_account_email_address>: A valid user account
+# <FIXME:service_account_email_address>: A valid service account email address
+# Refer to https://cloud.google.com/storage/docs/json_api/v1/buckets for more details
+
+imports:
+  - path: templates/gcs_bucket/gcs_bucket.py
+    name: gcs_bucket.py
+
+resources:
+  - name: test-bucket
```

Comment:
test-bucket needs to be changed to <FIXME:bucket_name> at every occasion to avoid failing by default.

Reply:
added `<FIXME:bucket_name>` to all example files. Updates in commit `8969659`